website/contributing/release-testing.md

```diff
@@ -35,100 +48,64 @@ These steps needs to be followed by the release crew as part of the release proc
+To ensure that we cover the most use cases, we need to ensure we test all these different combination of configurations:
 
-In the `RNTester` you want to also play around with the app and try different components: some important onces are `Flatlist`, `Image` and the "New Architecture Component" (should be the very last one in the list).
+- RNTester + iOS + Hermes
+- RNTester + iOS + JSC
+- RNTester + Android + Hermes
+- RNTester + Android + JSC
+- RNTestProject + iOS + Hermes
+- RNTestProject + iOS + JSC
+- RNTestProject + Android + Hermes
+- RNTestProject + Android + JSC
 
-An important element of testing source in the release branch is that this process needs to be preferably be done twice, on two separate machines - there are multiple scenarios in which something might fail on a machine but not on another. By running tests on two computers, we want to reach a step of "double confirmation" that everything works fine.
-
-## Testing an RC on a project - checklist
-
-If you are a [release tester](./release-roles-responsibilities#release-tester-responsibilities), the ask for you is to set as dependency in your app the latest RC available and report in the related "Road to 0.XX" how it went ([example](https://github.com/reactwg/react-native-releases/discussions/26)). To help provide the relevant information, we have prepared this template: you can copy/pasted it in a comment and fill it accordingly.
-
-```markdown
-| Link to branch:              |                         |
-| ---------------------------- | :---------------------- |
-| **Project info**             |                         |
-| Name                         |                         |
-| Starting RN version          | <add me, ex. RN 0.65.1> |
-| Hermes on iOS                | yes/no                  |
-| Hermes on Android            | yes/no                  |
-| **Tested - iOS**             |                         |
-| Fast Refresh                 | ✅/🚨/🙅‍♂️                |
-| Debug/dev build on Simulator | ✅/🚨/🙅‍♂️                |
-| Debug/dev build on Device    | ✅/🚨/🙅‍♂️                |
-| Production build             | ✅/🚨/🙅‍♂️                |
-| Chrome remote debugger       | ✅/🚨/🙅‍♂️                |
-| Hermes debugger              | ✅/🚨/🙅‍♂️                |
-| Flipper debugger             | ✅/🚨/🙅‍♂️                |
-| Deploy to TestFlight         | ✅/🚨/🙅‍♂️                |
-| **Tested - Android**         |                         |
-| Fast Refresh                 | ✅/🚨/🙅‍♂️                |
-| Debug/dev build on Emulator  | ✅/🚨/🙅‍♂️                |
-| Debug/dev build on Device    | ✅/🚨/🙅‍♂️                |
-| Production build             | ✅/🚨/🙅‍♂️                |
-| Chrome remote debugger       | ✅/🚨/🙅‍♂️                |
-| Hermes debugger              | ✅/🚨/🙅‍♂️                |
-| Flipper debugger             | ✅/🚨/🙅‍♂️                |
```

Comment:
I'm a bit afraid on losing this list, which I believe it's still valuable to keep in some form, especially if we onboard external QA in some form

Reply:
I have it saved in my notes, we can bring it back if needed when the QA ramps up?

Reply:
the thing is that it's aimed to someone testing RN against a prod app, so I don't know how valuable it'd be to have in the docs here

Reply:
> I have it saved in my notes, we can bring it back if needed when the QA ramps up?

I don't have a strong opinion. We can remove it for now and bring it back if needed

Reply:
@cortinico readded in its own dedicated section so that we can link it in the "road to" discussion